pilot/pkg/security/authz/model/model.go

```diff
@@ -34,7 +35,7 @@ const (
 	RBACTCPFilterStatPrefix = "tcp."
 
 	// attributes that could be used in both ServiceRoleBinding and ServiceRole.
-	attrRequestHeader = "request.headers" // header name is surrounded by brackets, e.g. "request.headers[User-Agent]".
+	attrRequestHeader = "request.headers" // header name is surrounded by brackets, e.g. "request.headers[Users-Agent]".
```

Comment:
I think this comment should not be changed.

Reply:
Wrong git add, thanks.